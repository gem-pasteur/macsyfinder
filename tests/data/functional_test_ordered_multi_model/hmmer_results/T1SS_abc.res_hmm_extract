# gene: T1SS_abc extract from functional_test_ordered_multi_model/hmmer_results/T1SS_abc.search_hmm.out hmm output
# profile length= 501
# i_evalue threshold= 0.001
# coverage threshold= 0.500
# hit_id replicon_name position_hit hit_sequence_length gene_name gene_system i_eval score profile_coverage sequence_coverage begin end
VICH001.B.00001.C001_01399	UserReplicon	10	720	T1SS_abc	1.900e-158	519.700	1.000	0.699	167	669
