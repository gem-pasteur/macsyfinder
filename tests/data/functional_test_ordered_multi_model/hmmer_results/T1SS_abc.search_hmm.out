# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.1b2 (February 2015); http://hmmer.org/
# Copyright (C) 2015 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  tests/data/models/functional/profiles/T1SS_abc.hmm
# target sequence database:        tests/data/base/test_14.fasta
# output directed to file:         functional_test_ordered_multi_model/hmmer_results/T1SS_abc.search_hmm.out
# sequence reporting threshold:    E-value <= 0.1
# number of worker threads:        0
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       abc  [M=501]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence                   Description
    ------- ------ -----    ------- ------ -----   ---- --  --------                   -----------
   1.6e-158  520.0   9.5   1.9e-158  519.7   9.5    1.1  1  VICH001.B.00001.C001_01399  abc


Domain annotation for each sequence (and alignments):
>> VICH001.B.00001.C001_01399  abc
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  519.7   9.5  4.3e-160  1.9e-158       1     501 []     167     669 ..     167     669 .. 1.00

  Alignments for each domain:
  == domain 1  score: 519.7 bits;  conditional E-value: 4.3e-160
                         abc   1 lsvvinllaLvsplymlqVyDrvlpsrnvstLavltvlllvlilleallevlRslvvvrlgskldvelnarvfkallelpl.. 81 
                                 +s+v+++laL+spl++++V+D+vl++++ stL vl   l++  ++e +l +lR +  ++++ ++d++l+ ++ ++l+ lpl  
  VICH001.B.00001.C001_01399 167 FSFVLQILALISPLFFQVVMDKVLVHQAWSTLDVLVFGLVITGVIEVVLRGLREYQYAHTANRIDIQLGLKLVQHLFGLPLmf 249
                                 79********************************************************************************* PP

                         abc  82 .ekrrkgdagqrlrdldqlRafltgsaltallDapwvvvflvvvflyhpklgllvlagllvlialslvteavlrkrleeklkr 163
                                  ++r++g +++r+r+ld++R+fltgs  t  ++++++ vfl v+ l+++ l+ l +a+++++++l+   ++ ++  +e+++++
  VICH001.B.00001.C001_01399 250 fKSRQVGAIVTRVRELDTIREFLTGSMFTLTVELLFMFVFLYVMSLLSAPLTGLFIATVPCYVLLAWWLTPRMQAAIEKQFSH 332
                                 *********************************************************************************** PP

                         abc 164 saeaqslavetlrnaevikalgmlgqlkrrWlrqkaeyvalqlkasevaaavasisklvraalqsliLglgallaidgkitiG 246
                                 +a+++s+++et+ ++e++k+l++++++ rrW++q +++v+++  + +++++ + +++l++++  + iL+lga+ +++ ++tiG
  VICH001.B.00001.C001_01399 333 AAANTSFLTETVAGSETLKSLAVEPRFIRRWDEQTEKMVTTGYDVQQLNNRSNHLVQLLQKITSVAILWLGATEVLSLEMTIG 415
                                 *********************************************************************************** PP

                         abc 247 mliAgsilvgrvlapveqligvwkqfssarlsleRledllkaekeakaklelpapegslsvegvslrykgdsadvlsnvslsi 329
                                 +liA+++++ ++ +p+++++++w qf ++r+++e+l+d+l+   e+++  + + ++g++s++++ +ry++d  ++++++sl i
  VICH001.B.00001.C001_01399 416 QLIAFNMMTNHIAQPLARMVELWGQFIQTRVAIEKLGDMLNLPVEQHTGSDNVTISGAISFKNILFRYQPDIPPTINDLSLDI 498
                                 *********************************************************************************** PP

                         abc 330 aaGevlgivGaSgsGKstLarllvglfepesGrvrldgadikqvdkeklgrqigyllQdieLfsgsvaeNIarfaeemdsekv 412
                                 +aGe+lg+vG+SgsGKstLarll +l++pe+G++ +dg+ ++ ++ ++l++ +g++lQ++ Lf++sv eNIa+ ++e++ e++
  VICH001.B.00001.C001_01399 499 RAGETLGVVGTSGSGKSTLARLLLRLYSPEQGSITIDGIPLNHINVQQLRQRVGVVLQENFLFHKSVSENIAQSKPEASLEEI 581
                                 *********************************************************************************** PP

                         abc 413 vaaAklagvHelIlelpeGYdtliGegGagLSgGQrQRialARAlyrkPallvlDEansaLDsegekalqraikklkargrtv 495
                                 ++aAkl+g+H++Il+lp+GYdt++ egG++LSgGQrQR+a+AR l+ +P++l+lDEa+saLD+e++ ++q+++ ++ argrtv
  VICH001.B.00001.C001_01399 582 IEAAKLSGAHDFILKLPMGYDTVLAEGGQSLSGGQRQRLAIARTLLSDPKVLILDEATSALDDESQAVIQANMASI-ARGRTV 663
                                 ****************************************************************************.****** PP

                         abc 496 vliaHr 501
                                 + iaHr
  VICH001.B.00001.C001_01399 664 ITIAHR 669
                                 *****9 PP



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                              1  (501 nodes)
Target sequences:                           45  (18710 residues searched)
Passed MSV filter:                         5  (0.111111); expected 0.9 (0.02)
Passed bias filter:                        5  (0.111111); expected 0.9 (0.02)
Passed Vit filter:                         2  (0.0444444); expected 0.0 (0.001)
Passed Fwd filter:                         1  (0.0222222); expected 0.0 (1e-05)
Initial search space (Z):                 45  [actual number of targets]
Domain search space  (domZ):               1  [number of targets reported over threshold]
# CPU time: 0.00u 0.00s 00:00:00.00 Elapsed: 00:00:00.00
# Mc/sec: inf
//
[ok]
